applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -0,0 +1,209 @@
+* `SLM proprietary command list #XCLAC`_
+* `Power saving #XSLEEP`_
+
+----
+
+SLM version #XSLMVER
+====================
+
+The ``#XSLMVER`` command requests the SLM version.
+
+Set command
+-----------
+
+The set command requests the SLM version.
+
+Syntax
+~~~~~~
+
+::
+
+   #XSLMVER
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XSLMVERSION: <version>
+
+The ``<version>`` value returns a string containing the SLM version followed by ``<CR><LF>``.
+
+Example
+~~~~~~~
+
+The following command example reads the SLM version:
+
+::
+
+   AT#XSLMVER
+   #XSLMVER: 1.2
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+----
+
+SLM proprietary command list #XCLAC
+===================================
+
+The ``#XCLAC`` command requests the list of the proprietary SLM commands.
+
+Set command
+-----------
+
+The set command requests the list of the proprietary SLM commands.
+It is an add-on for ``AT+CLAC``, which lists all modem AT commands.
+
+Syntax
+~~~~~~
+
+::
+
+   #XCLAC
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   <command list>
+
+The ``<command list>`` value returns a list of values representing all the ``#X*`` commands followed by <CR><LF>.
+
+Example
+~~~~~~~
+
+The following command example reads the SLM version: ::
+
+   at#xclac
+   AT#XSLMVER
+   AT#XSLEEP
+   AT#XCLAC
+   AT#XSOCKET
+   AT#XBIND
+   AT#XCONNECT
+   AT#XSEND
+   AT#XRECV
+   AT#XSENDTO
+   AT#XRECVFROM
+   AT#XPING
+   AT#XGPSRUN
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+----
+
+Power saving #XSLEEP
+====================
+
+The ``#XSLEEP`` command makes the nRF91 development kit go into idle or sleep mode.
+
+Set command
+-----------
+
+The set command requests the list of the proprietary SLM commands.
+
+Syntax
+~~~~~~
+
+::
+
+   #XSLEEP[=<shutdown_mode>]
+
+The ``<shutdown_mode>`` value accepts only the following integer values:
+
+* ``0`` - Enter Idle
+* ``1`` - Enter Sleep
+
+The default value is 0.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+* In case of Idle, it will exit by sending data over UART.
+* In case of Sleep, it will wake up by GPIO or reset.
+
+.. csv-table:: Parameters
+	:header: "Parameter", "Value type", "Accepted Values", "Default Value"
+	:widths: 10, 10, 30, 10
+
+	"``shutdown_mode``", "Integer", "``0`` - Enter Idle
+	``1`` - Enter Sleep", "``0``"
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XSLEEP
+
+::
+
+   AT#XSLEEP=0
+
+::
+
+   AT#XSLEEP=1
+
+::
+
+   AT#XSLEEP?
+   ERROR
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command tests the existence of the command and provides information about the type of its subparameters.
+
+Syntax
+~~~~~~
+
+::
+
+   #XSLEEP=?
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XSLEEP: <list of shutdown_mode>
+
+Example
+~~~~~~~
+
+::
+
```

Comment:
@lats1980 Is this example missing as 1st line "#XSLEEP=?" ?

Reply:
We have not listed READ and TEST commands in README before.

Reply:
@junqingzou Apparently the test command for XSLEEP is present in the excel file, both in the 1.2 and 1.4 version. Could you please check if this should be removed?